sc/src/main/webapp/WEB-INF/tiles/views.xml

```diff
@@ -10,8 +10,8 @@
             <definition template="/WEB-INF/jsp/tiles/views/home.jsp">
                 <put-attribute name="home-hero" value="/WEB-INF/jsp/tiles/views/home/hero.jsp"/>
                 <put-attribute name="search" value="/WEB-INF/jsp/tiles/views/home/search.jsp"/>
-                <put-attribute name="experiment-list-latest-box"
-                               value="/WEB-INF/jsp/tiles/views/home/experiment-list-latest.jsp"/>
+                <put-attribute name="experiment-and-species-panels"
```

Comment:
If you change the attribute name here you should change it in the corresponding view as well.